docs/contributing/build-vyos.rst

```diff
@@ -741,6 +741,38 @@ binaries in ``vyos-build/packages/linux-kernel`` from which you can copy them
 to the ``vyos-build/packages`` folder for inclusion during the ISO build.
 
 
+Mellanox OFED
+^^^^^^^^^
```

Comment:
please add some `^` to have the same length as the headline text

Reply:
Thanks for pinging, didn't notice this in the mail flood. Yessir, will get that done by weekend (on the move presently)